### ISSUE TITLE
If ECS service-linked role doesn't exist, CF Stack will rollback

### ISSUE THREAD

**Issue by chrisckchang** [awslabs/iot-device-simulator#1]
Hi,

Trying to provision the CF template without the ECS-generated service-linked role "AWSServiceRoleForECS" will result in a rollback. It would likely be helpful to highlight this requirement and include a pointer to the documentation for creating a service-linked role: https://docs.aws.amazon.com/IAM/latest/UserGuide/using-service-linked-roles.html#create-service-linked-role.

Quick research suggests that CloudFormation may not be able to elegantly create this role, so not sure what's the best way to proceed. Happy to help if given guidance.

**Comment by shsenior** [comment]
@chrisckchang You are absolutely correct. We updated the README and we will get this documented in the implementation guide.
